docs/scenarios_authoring/Spel.md

```diff
@@ -259,16 +265,25 @@ numeric types and between `String` and some useful logical value types. Some exa
 | `'USD'`                                  | String     | Currency               |
 | `'bf3bb3e0-b359-4e18-95dd-1d89c7dc5135'` | String     | UUID                   |
 
-You can also use explicit conversions that are available in utility classes and build-in java conversion mechanisms:
+Usage example:
+
+| Expression                         | Input value       | Input type | Target type |
+|------------------------------------|-------------------|------------|-------------|
+| #DATE.now.atZone('Europe/Warsaw')  | `'Europe/Warsaw'` | String     | ZoneId      |
+
+More usage examples can be found in [this section](#conversions-between-datetime-types).
+
+#### Explicit conversions
 
-| Expression                                                      | Result                    | Type           |
-| ------------                                                    | --------                  | --------       |
-| `#NUMERIC.toNumber('42')`                                       | 42                        | Number         |
-| `#NUMERIC.toNumber('42').toString()`                            | '42'                      | String         |
-| `#DATE_FORMAT.parseOffsetDateTime('2018-10-23T12:12:13+00:00')` | 1540296720000             | OffsetDateTime |
-| `#DATE_FORMAT.parseLocalDateTime('2018-10-23T12:12:13')`        | 2018-10-23T12:12:13+00:00 | LocalDateTime  |
-| `'' + 42`                                                       | '42'                      | String         |
+Explicit conversions are available in utility classes and build-in java conversion mechanisms:
 
+| Expression                                                      | Result                     | Type            |
+|-----------------------------------------------------------------|----------------------------|-----------------|
+| `#NUMERIC.toNumber('42')`                                       | 42                         | Number          |
+| `#NUMERIC.toNumber('42').toString()`                            | '42'                       | String          |
+| `#DATE_FORMAT.parseOffsetDateTime('2018-10-23T12:12:13+00:00')` | 1540296720000              | OffsetDateTime  |
+| `#DATE_FORMAT.parseLocalDateTime('2018-10-23T12:12:13')`        | 2018-10-23T12:12:13+00:00  | LocalDateTime   |
+| `'' + 42`                                                       | '42'                       | String          |
```

Comment:
Just wondering - this may be an implicit conversion (?)

Reply:
Done